pkg/api/deletetask.go

```diff
@@ -0,0 +1,28 @@
+package api
+
+import (
+	"database/sql"
+	"net/http"
+
+	"github.com/rizhyi/final-sprint/pkg/db"
+)
+
+func deleteTaskHandler(w http.ResponseWriter, r *http.Request) {
+	id := r.URL.Query().Get("id")
+	if id == "" {
+		writeJSON(w, map[string]string{"error": "Не указан идентификатор"})
+		return
+	}
+
+	err := db.DeleteTask(id)
+	if err != nil {
+		if err == sql.ErrNoRows {
```

Comment:
Лучше так сравнивать
```suggestion
		if errors.Is(err, sql.ErrNoRows) {
```